incubator/kafka/charts/zookeeper/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+name: zookeeper
```

Comment:
Helm now supports specifying dependencies using a `requirements.yaml` file. Can you update this to remove this vendored version of zookeeper, and instead just add a requirements.yaml that can be used to bring in the dependency? 

See https://github.com/kubernetes/helm/blob/master/docs/charts.md#managing-dependencies-with-requirementsyaml and https://github.com/kubernetes/charts/tree/master/incubator/patroni for an example.